src/workerd/api/worker-rpc.c++

```diff
@@ -1731,7 +1733,35 @@ class EntrypointJsRpcTarget final: public JsRpcTargetBase {
           "\"cloudflare:workers\".");
     }
 
-    TargetInfo targetInfo{.target = jsg::JsObject(handler->self.getHandle(lock)),
+    auto target = jsg::JsObject(handler->self.getHandle(lock));
+
+    KJ_IF_SOME(moduleName, wrapperModule) {
+      // We've been asked to apply a wrapper module to the handler. This is a builtin module whose
+      // default export is a function. The function is invoked with the argument being the DO
```

Comment:
I guess the default export has been updated to be a class instead of a function?  (But I'm not actually sure if there's a significant difference in JS.)